docs/quick-start.md

```diff
@@ -22,42 +23,41 @@ import {
 ```
 
 ## 3. Get some _AE_ using the Faucet
-To receive some _AE_ you can use the [Faucet](https://faucet.aepps.com/). Just add your publicKey, and you'll immediately get some test coins.
+To receive some _AE_ you can use the [Faucet](https://faucet.aepps.com/). Just paste sender's publicKey, hit `Top UP` and you'll immediately get some test coins.
 
 ## 4. Interact with the æternity blockchain
 This example shows:
 
 - how to create an instance of the SDK using the `Universal` [Stamp](https://stampit.js.org/essentials/what-is-a-stamp)
-- how to get the current block height
-- how to spend 1 AE from the account the SDK instance was initialized with to some other AE address
+- how to spend (send) 1 AE from the account the SDK instance was initialized with to some other AE address
 
 ```js
 const NODE_URL = 'https://testnet.aeternity.io';
-const COMPILER_URL = 'https://compiler.aepps.com'; // required for contract interactions (compiling and creating calldata)
-const account = MemoryAccount({
-  // replace <YOUR_SECRET> and <YOUR_PUBLIC_KEY> with the values from step 2
-  keypair: { secretKey: '<YOUR_SECRET>', publicKey: '<YOUR_PUBLIC_KEY>' }
+const COMPILER_URL = 'https://compiler.aepps.com'; // required for contract interactions
+const senderAccount = MemoryAccount({
+  // replace <SENDER_SECRET> and <SENDER_PUBLIC_KEY> with the generated keyppair from step 2
+  keypair: { secretKey: '<SENDER_SECRET>', publicKey: '<SENDER_PUBLIC_KEY>' }
 })
 
 (async function () {
   const node = await Node({ url: NODE_URL })
   const aeSdk = await Universal({
      compilerUrl: COMPILER_URL,
      nodes: [ { name: 'testnet', instance: node } ],
-     accounts: [ account ]
+     accounts: [ senderAccount ]
   })
 
-  const height = await aeSdk.height() // get top keyblock height
-  console.log('Current Keyblock Height:', height)
-
   // spend one AE
-  await aeSdk.spend(1, '<RECIPIENT_PUBLIC_KEY>', { // replace <RECIPIENT_PUBLIC_KEY>
+  await aeSdk.spend(1, '<RECIPIENT_PUBLIC_KEY>', {
+      // replace <RECIPIENT_PUBLIC_KEY>, Ideally you use public key from Superhero Wallet you have created before
```

Comment:
Here I wanted to point developers to set up Superhero.

Otherwise we would need to guide developers to generate another (recipient) account.
Make sense to me as this is the very first example.

Reply:
```suggestion
    // replace <RECIPIENT_PUBLIC_KEY>, Ideally you use public key from Superhero Wallet you have created before
```
the indentation should be 2 spaces as I see

Reply:
fixed